plugins/newsdownloader.koplugin/main.lua

```diff
@@ -160,12 +160,13 @@ function NewsDownloader:loadConfigAndProcessFeeds()
     for idx, feed in ipairs(feed_config) do
         local url = feed[1]
         local limit = feed.limit
+        local createNewsFromDescription = feed.createNewsFromDescription == "true" or feed.createNewsFromDescription == true
```

Comment:
Functions are `likeThis`, variables are `like_this`. That way when you're reading the code you know what's what and it's easier to spot mistakes.

Also I think it should be something much more logical like `download_full_page`.

Reply:
Java habits sorry:P